versioned_docs/version-8.1/self-managed/identity/user-guide/configure-logging.md

```diff
@@ -1,22 +1,104 @@
 ---
 id: configure-logging
-title: "Configuring logging"
-sidebar_label: "Configuring logging"
+title: "Configure logging"
+sidebar_label: "Configure logging"
+description: "In this guide we will demonstrate how to configure logging in the Identity component"
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+## Configuring logging
+
 The Identity component uses the [Log4j2](https://logging.apache.org/log4j/2.x/) framework to control
 the log level and log format.
 
-By default, Identity logs at the `info` and produces a log line with following format:
-
-```
-2022-06-27 15:05:21.442 INFO 34490 --- [main] i.c.i.Application: Started Application in 3.659 seconds (JVM running for 4.185)
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN" monitorInterval="30">
+    <Properties>
+        <Property name="LOG_PATTERN">%clr{%d{yyyy-MM-dd HH:mm:ss.SSS}}{faint} %clr{%5p} %clr{${sys:PID}}{magenta}
+            %clr{---}{faint} %clr{[%15.15t]}{faint} %clr{%-40.40c{1.}}{cyan} %clr{:}{faint} %m%n%xwEx
+        </Property>
+        <Property name="LOG_FILE_PATTERN">%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{1.} %enc{%msg}%n
+        </Property>
+        <Property name="LOG_FILE_NAME_PATTERN">logs/identity.%d{yyyy-MM-dd-mm-ss}.log</Property>
+    </Properties>
+    <Appenders>
+        <Console name="Console" target="SYSTEM_OUT" follow="true">
+            <PatternLayout pattern="${env:IDENTITY_LOG_PATTERN:-${LOG_PATTERN}}"/>
+        </Console>
+        <Console name="Stackdriver" target="SYSTEM_OUT" follow="true">
+            <JsonTemplateLayout eventTemplateUri="classpath:GcpLayout.json" locationInfoEnabled="true"/>
+        </Console>
+        <RollingFile
+                name="File"
+                fileName="${env:IDENTITY_LOG_FILE_NAME:-logs/identity.log}"
+                filePattern="${env:IDENTITY_LOG_FILE_NAME_PATTERN:-${LOG_FILE_NAME_PATTERN}}"
+                append="true">
+            <PatternLayout pattern="${env:IDENTITY_LOG_FILE_PATTERN:-${LOG_FILE_PATTERN}}"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy interval="${env:IDENTITY_LOG_FILE_ROTATION_DAYS:-1}"/>
+                <SizeBasedTriggeringPolicy size="${env:IDENTITY_LOG_FILE_ROTATION_SIZE:-50 MB}"/>
+            </Policies>
+        </RollingFile>
```

Comment:
@Ben-Sheppard if this is referenced as what we include per default, then `File` logger is not included on 8.1